### ISSUE TITLE
Improved set-up instructions: gistId and personalAccessToken

### ISSUE THREAD

**Issue by solsend2l** [Hackafe/atom-sync-settings#161]
Thanks for the great plugin! I think it would help to spell out these exact keynames in the Atom `config.cson` in README.md:

```
  "sync-settings":
    gistId: "b302...88c46c"
    personalAccessToken: "7aa0cc2...a1e7e8641"
```

I had to search through the code base to find them.

**Comment by richbs** [comment]
Thanks for this tip @solsend2l, most helpful!

**Comment by BrianSipple** [comment]
+1 

Thanks, @solsend2l!

**Comment by dirk-thomas** [comment]
That sounds like a good improvement. Can you please provide a PR for this.

**Comment by solsend2l** [comment]
Done!

**Comment by dirk-thomas** [comment]
Addressed by #166. Thanks!
